### PR TITLE
Updated README with how to use T2 Key Vault SDK.

### PR DESCRIPTION
**Pull request by FrankieTF**
Updated : README.
- Adding how to use latest Key Vault SDK in this sample.

### CODE REVIEW

**Comment by jongio** [comment]
@heaths - Can you please review?  Thanks

File: README.md [hunk 1]
```diff
@@ -29,6 +29,17 @@ Support for Azure Key Vault-managed storage account may be colloquially referred
 * List and get existing managed storage SAS definitions
 * Obtain a SAS token from Azure Key Vault and use it to access an Azure Storage endpoint
 
+## Use latest Key Vault SDK
```
Comment (heaths) [inline README.md:32]:
```suggestion
## Use the latest Key Vault SDK
```

File: README.md [hunk 2]
```diff
@@ -29,6 +29,17 @@ Support for Azure Key Vault-managed storage account may be colloquially referred
...
+
+The Key Vault SDK in this repo for Azure Key Vault manages storage accounts is based on Microsoft.Azure.KeyVault. Now it's recommended to use [role-based access control (RBAC)](https://docs.microsoft.com/azure/role-based-access-control/overview) to secure access to your storage accounts.
```
Comment (heaths) [inline README.md:34]: Use a conversational tone per Microsoft writing guidelines:
```suggestion
The Key Vault SDK previously used to manage storage accounts is based on Microsoft.Azure.KeyVault. We now recommended you use [role-based access control (RBAC)](https://docs.microsoft.com/azure/role-based-access-control/overview) to secure access to your storage accounts.
```

File: README.md [hunk 3]
```diff
@@ -29,6 +29,17 @@ Support for Azure Key Vault-managed storage account may be colloquially referred
...
+[ShareLink](https://docs.microsoft.com/samples/azure/azure-sdk-for-net/share-link/) demonstrates how to generate a client library for Azure Key Vault manages storage accounts based on latest Key Vault SDK.
+
+[Notes] This sample needs permissions deletesas, recover, backup, restore update purge, so step 4 of **Getting Start** in [ShareLink](https://docs.microsoft.com/samples/azure/azure-sdk-for-net/share-link/) should be updated to the command like below in Azure CLI:
```
Comment (heaths) [inline README.md:37]:
```suggestion
Note that this sample needs permissions `deletesas`, `recover`, `backup`, `restore`, `update`, and  `purge`, so step 4 of **Getting Start** in [ShareLink](https://docs.microsoft.com/samples/azure/azure-sdk-for-net/share-link/) should be updated using the Azure CLI to the following command:
```

**Review by heaths — APPROVED** [review]

**Pull request merged**